calculator.c

```diff
@@ -0,0 +1,54 @@
+://Clarke Littlejohn
+//A simple five function calculator
```

Comment:
Assuming you didn't make this small error though, your code compiled! You didn't handle floats as you were supposed to, however.